packages/pixeloven-core/common/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "extends": "../../tsconfig",
+    "extends": "../../../tsconfig",
```

Comment:
Any way to add a custom path to avoid having to change this if the organization changes again?

Reply:
Not that I can find. Eventually I want to overhaul the whole config setup but this is a built in function of TS itself